modules/service/src/main/scala/lucuma/odb/graphql/input/SetProposalStatusInput.scala

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.odb.graphql.input
+
+import cats.syntax.all.*
+import lucuma.core.model.Program
+import lucuma.core.model.ProgramReference
+import lucuma.core.model.ProposalReference
+import lucuma.odb.data.Tag
+import lucuma.odb.graphql.binding.*
+
+final case class SetProposalStatusInput(
+  programId: Option[Program.Id],
+  proposalReference: Option[ProposalReference],
+  programReference: Option[ProgramReference],
+  status:           Tag
```

Comment:
I hope we can switch up our enum handling somehow.

Reply:
Yeah, me too.